src/app/courses/components/edit-course-page/edit-course-page.component.ts

```diff
@@ -16,29 +17,46 @@ import { State } from '../../../reducers';
 })
 export class EditCoursePageComponent implements OnInit, OnDestroy {
   public id: string;
-  public name: string;
-  public description: string;
-  public date: any;
-  public length: number;
-  public currentCourse: ICourseItem;
   private sub$: Subscription;
+  private maxTitle = 50;
+  private maxDescription = 500;
+  private authors: any;
```

Comment:
Also, you shouldn't use `any` if you know object structure, because you loose all the benefits of static type system when you do it. Typescript is no longer able to tell you when you doing something illegal. So it would be much better to define Author interface and use it instead here: `authors: Author[]`

Reply:
done